### ISSUE TITLE
Help to configure WebRTC for Dahua doorbell 

### ISSUE THREAD

**Issue by omenukas** [AlexxIT/go2rtc#476]
Hello,
I have a Dahua VTO3211D-P2-S2 doorbell. Could someone please help with the integration to HA?
HA has WebRTC Camera, go2rtc, Onvif and Generic camera installed, but I don't understand how to configure everything correctly for webrtc.
Generic camera and Onvif works, but webRTC doesn't even show the video:
![image](https://github.com/AlexxIT/go2rtc/assets/122688908/f9080e56-b6e7-4fa9-bfb4-c26a933fa11c)

Card Configuration
type: custom:webrtc-camera
url: rtsp://admin:pass!@192.168.32.99:554/cam/realmonitor?channel=1&subtype=0&unicast=true&proto=Onvif

Since I don't really understand it, I wrote only one in go2rtc.yaml:

stream:
  intercom_go2rtc: rtsp://admin:pass@192.168.32.99:554/cam/realmonitor?channel=1&subtype=0&unicast=true&proto=Onvif

I would really like to use the doorbell on a wall-mounted tablet and I really don't want to buy a VTH monitor, but I really lack the knowledge to implement it. I've seen audio issues resolved in other threads, but in my case, even the video on the WEBRTC card doesn't work.

**Comment by AlexxIT** [comment]
You have very strange error - `'webrtc'`
I don't know why it could happen.

**Comment by Cr4z33** [comment]
@omenukas I've got an Amcrest AD410 (this company shares same company like Dahua and Hikvision) and I don't use the `&unicast=true&proto=Onvif` part.

Give it a try?

**Comment by omenukas** [comment]
> @omenukas I've got an Amcrest AD410 (this company shares same company like Dahua and Hikvision) and I don't use the `&unicast=true&proto=Onvif` part.
> 
> Give it a try?

I also tried this option:
type: custom:webrtc-camera
url: rtsp://admin:pass@192.168.32.99:554/cam/realmonitor?channel=1&subtype=1

Some...

**Comment by AlexxIT** [comment]
Do you have some other settings in your card?
What is your browser? What is your OS?

**Comment by omenukas** [comment]
> Do you have some other settings in your card? What is your browser? What is your OS?

No, no other settings are on the card. The computer's OS is Windows 10. I tried with Crome, Edge, and Firefox browsers.

**Comment by omenukas** [comment]
if I choose a stream from the go2rtc panel,
![image](https://github.com/AlexxIT/go2rtc/assets/122688908/361a2abb-bf59-42c8-ab30-94475e65ac57)

 then the video shows:
![image](https://github.com/AlexxIT/go2rtc/assets/122688908/69553621-c593-4251-9565-bd5b281885c4)

go2rtc.yaml
stream:
  intercom_go2rtc: rtsp://admin:pass@192.168.32.99:554/cam/realmonitor?channel=1&subtype=1

configuration.yaml
media_player:
  - platform: onkyo
    sources:
      video2: 'CBL/SAT'
      video3: 'GAME'
      video4: 'AUX'
      video6: 'HTPC'
      dvd: 'BD/DVD'
      phono: 'PHONO'
      cd: 'CD'
      fm: 'FM'
      usb: 'USB'
      network: 'INTERNET'
      strm-box: 'STRMBOX'
  - platform: webrtc
    name: Dahua Camera
    stream: dahua
    audio: pcmu/48000
  - platform: webrtc
    name: Tapo Camera
    stream: tapo
    audio: pcma
shell_command:
  onkyo_command: onkyo {{ cmd }}

I didn't do anything else

**Comment by TheDuke427** [comment]
Try adding "&subtype=0&authbasic=64" to the end of your URL.
